src/test/java/io/percy/appium/metadata/IosMetadataTest.java

```diff
@@ -173,8 +173,106 @@ public void testOrientatioAuto() {
     @Test
     public void testScaleFactor() {
         Cache.CACHE_MAP.clear();
-        Map details = new HashMap<>();
-        details.put("pixelRatio", 2);
-        Assert.assertEquals(metadata.scaleFactor().intValue(), pixelRatio.intValue());
+        when(driver.manage()).thenReturn(new Options() {
+
+          @Override
+          public void addCookie(Cookie cookie) {
+            // TODO Auto-generated method stub
+            throw new UnsupportedOperationException("Unimplemented method 'addCookie'");
+          }
+
+          @Override
+          public void deleteCookieNamed(String name) {
+            // TODO Auto-generated method stub
+            throw new UnsupportedOperationException("Unimplemented method 'deleteCookieNamed'");
+          }
+
+          @Override
+          public void deleteCookie(Cookie cookie) {
+            // TODO Auto-generated method stub
```

Comment:
why this?

Reply:
this is auto generated. since we are using driver.manage.window.size in pixel ratio.